WorkloadManagementSystem/DB/JobLoggingDB.py

```diff
@@ -50,38 +50,27 @@ def addLoggingRecord(self,
     event = 'status/minor/app=%s/%s/%s' % (status, minor, application)
     self.log.info("Adding record for job ", str(jobID) + ": '" + event + "' from " + source)
 
-    if not date:
-      # Make the UTC datetime string and float
-      _date = Time.dateTime()
-      epoc = time.mktime(_date.timetuple()) + _date.microsecond / 1000000. - MAGIC_EPOC_NUMBER
-      time_order = round(epoc, 3)
-    else:
-      try:
-        if isinstance(date, six.string_types):
-          # The date is provided as a string in UTC
-          _date = Time.fromString(date)
-          epoc = time.mktime(_date.timetuple()) + _date.microsecond / 1000000. - MAGIC_EPOC_NUMBER
-          time_order = round(epoc, 3)
-        elif isinstance(date, Time._dateTimeType):
-          _date = date
-          epoc = time.mktime(_date.timetuple()) + _date.microsecond / 1000000. - \
-              MAGIC_EPOC_NUMBER  # pylint: disable=no-member
-          time_order = round(epoc, 3)
-        else:
-          self.log.error('Incorrect date for the logging record')
-          _date = Time.dateTime()
-          epoc = time.mktime(_date.timetuple()) - MAGIC_EPOC_NUMBER
-          time_order = round(epoc, 3)
-      except BaseException:
-        self.log.exception('Exception while date evaluation')
+    try:
+      if not date:
+        # Make the UTC datetime string and float
+        _date = Time.dateTime()
+      elif isinstance(date, six.string_types):
+        # The date is provided as a string in UTC
+        _date = Time.fromString(date)
+      elif isinstance(date, Time._dateTimeType):
+        _date = date
+      else:
+        self.log.error('Incorrect date for the logging record')
         _date = Time.dateTime()
-        epoc = time.mktime(_date.timetuple()) - MAGIC_EPOC_NUMBER
-        time_order = round(epoc, 3)
+    except BaseException:
```

Comment:
```suggestion
    except Exception:
```

Reply:
Done but this makes `pylint` unhappy ;-)